src/models/degiroRecord.ts

```diff
@@ -8,5 +8,17 @@ export class DeGiroRecord {
     fx: string;
     currency: string;
     amount: string;
+    balance_currency: string; // not used, but improtant for hashing
+    balance: string; //// not used, but improtant for hashing
     orderId: string;
+
```

Comment:
The test are also breaking due to you using Number instead of number. 

See https://github.com/dickwolff/Export-To-Ghostfolio/actions/runs/12614130746/job/35162022748

Reply:
no problem. Once again. I had never used ts before, so I didn't know the difference. I'll switch to `number`